dashboard/www/scripts/models/plugin.js

```diff
@@ -15,9 +15,9 @@ window.discotron.Plugin = class extends window.discotron.PluginModel {
      * @param {boolean} options.enabled True if plugin is enabled by the bot owner
      * @param {Array} options.logs List of logs the plugin can output to
      */
-    constructor({pluginId, name, description, version, prefix, commands, defaultPermission, enabled, logs}) {
```

Comment:
I don't have an IDE to check right now, but don't you have to update some other references too? (Only did a quick CTRL+F thru the code, don't judge)... Such as `dashboard/backend/endpoints/plugin.js`
`dashboard/backend/endpoints/guild/plugin.js`
`dashboard/backend/endpoints/plugin/log.js`
`dashboard/www/scripts/controllers/widgets/plugin-settings-widget-controller.js`

Reply:
The only constructor for plugin I found properly uses `id`.
As far as I've checked, `pluginId` is only used when `id` would be ambiguous to indicate a plugin id (which means the code should've properly been updated to reflect this change)